packages/sak-visittkort/src/components/VisittkortPanel.tsx

```diff
@@ -97,6 +99,16 @@ const VisittkortPanel: FunctionComponent<OwnProps> = ({
               />
             </FlexColumn>
           )}
+          {relaterteFagsaker?.relaterteSøkere?.length > 0 &&
+            relaterteFagsaker.relaterteSøkere.map(relatertFagsak => (
+              <FlexColumn key={relatertFagsak.søkerIdent}>
+                <OtherPartCard
+                  fodselsnummer={relatertFagsak.søkerIdent}
+                  name={relatertFagsak.søkerNavn}
+                  url={`/k9/web/fagsak/${relatertFagsak.saksnummer}/`}
```

Comment:
dette må kanskje løses annerledes, men det fungerer

Reply:
Jeg syns det ser greit ut, men er kanskje bedre å bare sende ned saksnummeret som prop istedetfor url. Så kan `OtherPartCard` fikse urlen selv?

`paths.ts` i `sak-app/src/app` har også en egen `pathToFagsak`-funksjon du kan bruke for å lage url, så den skal du slippe å lage selv